compiler/natives/src/crypto/ed25519/ed25519vectors_test.go

```diff
@@ -0,0 +1,10 @@
+//go:build js
+// +build js
+
+package ed25519_test
+
+import "testing"
+
+func TestEd25519Vectors(t *testing.T) {
+	t.Skip("TODO") // runtime error: native function not implemented: syscall.runtime_BeforeFork
```

Comment:
Nitpick: I like to put the whole message ("runtime error: ...") into the t.Skip(), which makes it easy to see why the test was skipped just from the test log alone.

Also I think I've encountered referenced to BeforeFork previously and just defining an empty function seems to have worked.

Reply:
I left it as a TODO so I wouldn't forget to look more closely to see if it could be fixed.  But if it remains a Skip, I'll be sure to update.